### ISSUE TITLE
Question about different patch embedding implementation compared to official Wan-Video

### ISSUE THREAD

**Issue by ChenJiongxu355** [thu-ml/TurboDiffusion#114]
Hello,

Thank you for your great work!

While going through the code, I noticed that the implementation of patch_embedding differs from the official Wan-Video repository.

In [WanModel.py from Wan-Video](https://github.com/Wan-Video/Wan2.1/blob/main/wan/modules/model.py), patch embedding is implemented as:

`
self.patch_embedding = nn.Conv3d(
            in_dim, dim, kernel_size=patch_size, stride=patch_size)
`

However, in TurboDiffusion ([wan2pt1.py](https://github.com/thu-ml/TurboDiffusion/blob/main/turbodiffusion/rcm/networks/wan2pt1.py), it is implemented as:

`
self.patch_embedding = nn.Linear(in_dim * patch_size[0] * patch_size[1] * patch_size[2], dim)
`

I would like to understand the design choice behind replacing Conv3d with a Linear layer.

My main questions are:

What is the motivation for using nn.Linear instead of nn.Conv3d here?

If the pretrained Wan-Video model uses a Conv3d patch embedding, how are your patch embedding parameters trained to maintain model performance?

Since distillation only enforces consistency between teacher and student outputs, but does not guarantee that the output of your version will be consistent with that of official version.

I would really appreciate hearing from you.

**Comment by zhengkw18** [comment]
Linear and Conv3d are equivalent. You can refer to https://github.com/NVlabs/rcm/issues/16.
